### PR TITLE
Corpus pruning task: fix an exception in the result logging.

### PR DESCRIPTION
**Pull request by Dor1s**
https://pantheon.corp.google.com/errors/CN3g59XWt7TpcQ?time=P30D&project=clusterfuzz-external

//cc @oliverchang

### CODE REVIEW

File: src/python/bot/tasks/corpus_pruning_task.py [hunk 1]
```diff
@@ -502,17 +502,16 @@ def run(self, timeout):
                                            [self.context.shared_corpus_path],
                                            self.context.minimized_corpus_path,
                                            self.context.bad_units_path, timeout)
+      symbolized_output = stack_symbolizer.symbolize_stacktrace(result.logs)
```
Comment (inferno-chromium) [inline src/python/bot/tasks/corpus_pruning_task.py]: This code is not supposed to exception out, so can keep it at original place, but add return to the logs.log_error branch below (which causes the continue).

Comment (Dor1s) [reply src/python/bot/tasks/corpus_pruning_task.py]: Isn't it more readable in this form? IMO it's more straightforward to follow right now, but can change if you insist

**Pull request merged**